Civi/Angular/Manager.php

```diff
@@ -93,14 +77,15 @@ public function clear() {
    *     List of settings to preload.
    */
   public function getModules() {
-    if ($this->modules === NULL) {
+    $moduleNames = $this->cache->get('moduleNames');
```

Comment:
Are there still permission-guarded modules and dynamic-settings?

I think that `CRM_Mailing_Info::getAngularModules()` was substantively cleaned up a few years ago. (It still has the conditional - but that looks removable.)

In `universe`, these seem worth a look/think. (I grepped for `*ang.php` files that had conditionals and/or non-empty settings.)

```
./uk.co.vedaconsulting.mosaico/ang/crmMosaico.ang.php
./org.civicrm.civicase/ang/civicase.ang.php
./org.civicoop.areas/ang/areas.ang.php
./org.civicrm.volunteer/ang/volunteer.ang.php
./coop.symbiotic.timetrack/ang/timetrackImport.ang.php
./form-processor/ang/form_processor.ang.php
```

Reply:
Ah, it's all coming back to me now. That's why we did d56e4a996cf9de952db80c3823d572d2889365c0 - to prepare for this exact moment!

Reply:
@totten per our discussion I've [added noisy-deprecations and a status check](https://github.com/civicrm/civicrm-core/pull/27798) to core and submitted PRs/issues to all of the repos your `universe` search turned up, with the exception of CiviVolunteer (that one [already looks ok](https://github.com/colemanw/org.civicrm.volunteer/blob/master/ang/volunteer.ang.php)).